src/less/nault-theme.less

```diff
@@ -25,29 +25,84 @@ h1, h2, h3, h4, h5, .uk-text-lead, .uk-button, .uk-alert, .uk-description-list d
 	}
 
 	h2:not(.uk-card-title):not(.uk-modal-title) {
-	  	border-bottom: none!important;
+		border-bottom: none!important;
 	}
 }
 
 .uk-description-list dt {
-    font-weight: 700;
-    color: @global-color;
+	font-weight: 700;
+	color: @global-color;
```

Comment:
Did the spacing increase? Is this a local or repo setting?

Reply:
changed it to tabs to be consistent with my older .less files